### PR TITLE
Updated the section for cleanup rules

### PR DESCRIPTION
**Pull request by jpallavi**
Updated the section for "Automatically delete devices with cleanup rules" to add a note reflecting the latest permission required.

### CODE REVIEW

**Comment by traya1** [comment]
Learn Build status updates of commit _[9933a88](https://github.com/jpallavi/memdocs/commits/9933a8839a36467dfd3bee05f1426fa5ed68f593)_: 

### :bulb: Validation status: suggestions


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/remote-actions/devices-wipe.md](https://github.com/jpallavi/memdocs/blob/patch-1/memdocs/intune/remote-actions/devices-wipe.md) | :bulb:Suggestion | | [Details](#user-content-6fa2bbbcf058c3bacf900f690633b0c5-b49e07c2)

<a id="6fa2bbbcf058c3bacf900f690633b0c5-b49e07c2"></a>
### [memdocs/intune/remote-actions/devices-wipe.md](https://github.com/jpallavi/memdocs/blob/patch-1/memdocs/intune/remote-actions/devices-wipe.md)
 - **Line 195, Column 176**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/mem/intune/fundamentals/role-based-access-control' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202301101500134713-3494/BuildReport?accessString=338a9e4ec2164d5c350418e700d5d09dac9598a0b18db77d8a2b0352af56d544).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by jpallavi** [comment]
Updated the link to show relative path as per the suggestion above.

**Comment by traya1** [comment]
Learn Build status updates of commit _[c035ff5](https://github.com/jpallavi/memdocs/commits/c035ff507dc405313e4eeed2d60929ccdd6e2120)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-056ea802)

<a id="056ea802"></a>
###
 - **[Error: RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://aka.ms/SiteHelp and include the error report for our team to troubleshoot`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202301101507231454-3494/BuildReport?accessString=05ae0b56a8239b6938ff8373cfa5db0a7a2ebefa9d68e133763ebf52942c8cd8).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by traya1** [comment]
Learn Build status updates of commit _[c035ff5](https://github.com/jpallavi/memdocs/commits/c035ff507dc405313e4eeed2d60929ccdd6e2120)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/remote-actions/devices-wipe.md](https://github.com/jpallavi/memdocs/blob/patch-1/memdocs/intune/remote-actions/devices-wipe.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202301101555457419-3494/BuildReport?accessString=f41757b6b46c4bd8075b31b7fcc5dbd68ed1262368f67b7e240e5e176fef1b5d).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

File: memdocs/intune/remote-actions/devices-wipe.md [hunk 1]
```diff
@@ -191,6 +191,9 @@ The device clean up rule doesn't trigger a wipe or retire.
 > [!NOTE]
 > Device cleanup rules aren't available for Jamf-managed devices.
 
+> [!NOTE]
+> You need the permission **Managed Device Cleanup Settings with Update set to Yes** to be able to update the device cleanup rules. This permission is part of [Intune Roles](../fundamentals/role-based-access-control.md)
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/devices-wipe.md:195]:
```suggestion
>]
>> You need the permission **Managed Device Cleanup Settings with Update** set to **Yes** to be able to update the device cleanup rules. This permission is part of [Intune Roles](../fundamentals/role-based-access-control.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jpallavi, kindly commit the suggested change. Many thanks.

**Comment by jpallavi** [comment]
> Copy review - @jpallavi, kindly commit the suggested change. Many thanks.

Done, committed the suggested change, thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jpallavi** [comment]
Please suggest if this will be merged or if there is anything pending.

**Comment by JohanFreelancer9** [comment]
> Please suggest if this will be merged or if there is anything pending.

@yogkumgit, can you please assist? Thanks.

**Comment by tiburd** [comment]
@Smritib17 
Hi Smriti,
Please review this change.
If you approve but don't have perms to merge, please @ mention me to merge.
Thank you!

**Comment by jpallavi** [comment]
It is really critical for customers to see updated documentation, please suggest the path forward for merging this documentation.

**Pull request merged**